index.html

```diff
@@ -6,12 +6,55 @@
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300&display=swap" rel="stylesheet"> 
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@600&family=Raleway:wght@300&display=swap" rel="stylesheet">
+   <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@600&family=Raleway:wght@300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./css/style.css"/>
+   <link rel="stylesheet" href="./css/font-awesome.min.css">
+   <link rel="stylesheet" href="./css/media.css"/>
     <title>PalArt Gallery</title>
 </head>
 <body>
-
+    <!-- ****************************************************************************** -->
+ <!-- Contact Us section -->
+ <div class="contact-us">
```

Comment:
use semantic element 
there's a lot of divs, try to replace them with the semantic element